app/src/main/java/co/develhope/meteoapp/today/TodayFragment.kt

```diff
@@ -0,0 +1,289 @@
+package co.develhope.meteoapp.today
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatDelegate
+import co.develhope.meteoapp.R
+import co.develhope.meteoapp.databinding.FragmentHomeBinding
+import co.develhope.meteoapp.databinding.FragmentTodayBinding
+class TodayFragment : Fragment() {
+    private var _binding: FragmentTodayBinding? = null
+    private val binding get() = _binding!!
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentTodayBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.todayscreen.adapter = TodayAdapter(
+            listOf(
+                TodayData(
```

Comment:
da dove arrivano i dati? Dovresti avere una classe repository o più semplicemente una classe object Data, da cui leggi i dati relativi al meteo che poi usi per generare  la lista (che rappresenta la ui) da dare in pasto all'adapter